README.md

```diff
@@ -15,7 +15,6 @@ helm repo update
 
 ```shell
 helm install edge mezmo/edge \
-    --set mezmoApiAccessToken=MY_PIPELINE_SERVICE_TOKEN \
-    --set service.sourcePorts.start=8080 \
-    --set service.sourcePorts.end=8100
+  --namespace <MY_NAMESPACE> \
+  --set mezmoApiAccessToken=<MY_PIPELINE_SERVICE_TOKEN> \
```

Comment:
nit: for consistency with the edge readme, perhaps `MEZMO_API_ACCESS_TOKEN`

or maybe we should nix this and just link to the other readme from the root one